verse-pbc/plur
# fix new note not auto updating in group notes UI

## Issues covered
[#100](https://github.com/verse-pbc/issues/issues/100)

## Description
This PR fixes the issue where you post a new note in a group but the note is not instantly visible.

## How to test
1. Build the app.
2. Login and go to the communities screen.
3. Find a group to post a note in.
5. Click the + icon at the top bar of the group and write a note to post.
6. The note should be visible without you relaunching the app or refreshing.

## Video
Before:

https://github.com/user-attachments/assets/12462456-ed20-406b-839d-9ae68fecdc3a

After:

https://github.com/user-attachments/assets/ca069eab-9df9-40fa-a86b-ff040dc1b91d
